tests/test_main.py

```diff
@@ -191,7 +191,7 @@ def test_read_default_keywords_from_file(mocker):
             "files": [{"url": "http://diariooficial.feiradesantana.ba.gov.br/"}],
         }
     ]
-    expected_tweet = "Nele temos: decretos, aditivos, intimação"
+    expected_tweet = "Nele temos: decretos, aditivos"
```

Comment:
Como a variável foi adicionada, a saída do teste foi modificada também. Então temos duas alternativas:

1) Remover o conteúdo adicionado a variável de ambiente em `.env-example`
2) Deletar a variável de ambiente no início do teste com [monkeypatch](https://docs.pytest.org/en/6.2.x/monkeypatch.html): `monkeypatch.delenv(name, raising=False)`